packages/playground-ui/src/index.css

```
@@ -73,6 +73,20 @@
+  --toast-success-bg: color-mix(in srgb, var(--notice-success) 20%, var(--surface1));
+  --toast-success-border: color-mix(in srgb, var(--notice-success) 45%, var(--surface1));
+  --toast-destructive-bg: color-mix(in srgb, var(--notice-destructive) 20%, var(--surface1));
+  --toast-destructive-border: color-mix(in srgb, var(--notice-destructive) 45%, var(--surface1));
+  --toast-warning-bg: color-mix(in srgb, var(--notice-warning) 20%, var(--surface1));
+  --toast-warning-border: color-mix(in srgb, var(--notice-warning) 45%, var(--surface1));
+  --toast-info-bg: color-mix(in srgb, var(--notice-info) 20%, var(--surface1));
+  --toast-info-border: color-mix(in srgb, var(--notice-info) 45%, var(--surface1));
+
+  /* Toast shadow — dark mode skips the gloss highlight (white insets read muddy on the tinted
+     dark surface) and keeps only the elevation: tight close shadow + soft long shadow. */
+  --toast-shadow: 0 1px 2px oklch(0 0 0 / 0.4), 0 12px 32px -12px oklch(0 0 0 / 0.55);
```

Stylelint (17.9.0)

[error] Expected "0.4" to be "40%" (alpha-value-notation)

(alpha-value-notation)